static/app/views/starfish/views/webServiceView/spanGroupBreakdown.tsx

```diff
@@ -110,42 +113,51 @@ export function SpanGroupBreakdown({
             />
           )}
         </Header>
-        <Chart
-          statsPeriod="24h"
-          height={340}
-          showLegend
-          data={dataDisplayType === DataDisplayType.PERCENTAGE ? dataAsPercentages : data}
-          dataMax={dataDisplayType === DataDisplayType.PERCENTAGE ? 1 : undefined}
-          durationUnit={dataDisplayType === DataDisplayType.PERCENTAGE ? 0.25 : undefined}
-          start=""
-          end=""
-          errored={errored}
-          loading={isTimeseriesLoading}
-          utc={false}
-          onClick={handleModuleAreaClick}
-          grid={{
-            left: '0',
-            right: '0',
-            top: '20px',
-            bottom: '0',
-          }}
-          definedAxisTicks={6}
-          stacked
-          aggregateOutputFormat={
-            dataDisplayType === DataDisplayType.PERCENTAGE ? 'percentage' : 'duration'
-          }
-          tooltipFormatterOptions={{
-            valueFormatter: value =>
-              tooltipFormatterUsingAggregateOutputType(value, 'percentage'),
-          }}
-          onLegendSelectChanged={event => {
-            trackAnalytics('starfish.web_service_view.breakdown.legend_change', {
-              organization,
-              selected: Object.keys(event.selected).filter(key => event.selected[key]),
-              toggled: event.name,
-            });
-          }}
-        />
+        <VisuallyCompleteWithData
+          id={`WSV.${isEndpointBreakdownView ? 'EndpointBreakdown' : 'ServiceBreakdown'}`}
+          hasData={data.length > 0}
+        >
```

Comment:
I think it'd be okay to just call this `SpanGroupBreakdown`! You don't need the conditional IMO, it'd be fine to just have two separate widgets in a dashboard, each one filtering by the relevant transactions

Reply:
Yeah that's true! I think I coded this up before considering the dashboard, hence the conditional. I'll make this change